jekyll/_layouts/classic-docs.html

```diff
@@ -1,58 +1,138 @@
 <!DOCTYPE html>
 <html lang="{{ site.lang }}">
-{% include_cached head.html site=site page=page %}
+{% include head.html %}
 <body class="{% if page.page-type == "index" or page.page-type == "homepage" %}homepage{% endif %}">
 {% if jekyll.environment == "production" %}
-  {% include_cached google-tag-manager.html %}
+	{% include google-tag-manager.html %}
```

Comment:
Same here about `include_cached`. Also indentation level looks like it might be off?